### DISCUSSION TITLE
Any Way to Send Multipart Requests Without Attaching Files?

### DISCUSSION THREAD

**Discussion by laipz8200** [encode/httpx#3395] [Q&A] [answered]
I’ve reviewed the code and noticed that currently, a multipart request can only be sent when files is non-empty. Could we consider relaxing this restriction? For example, we could determine the need for a multipart request based on whether files is None rather than its emptiness.

I believe there should be a way to send data using multipart mode (for instance, simulating Postman’s behavior).

**Comment by lovelydinosaur** [comment, marked as answer]
> For example, we could determine the need for a multipart request based on whether files is None rather than its emptiness.

The seems like a smart change, yep.

**Reply by laipz8200** [reply]
Should I open an issue and then make this change?

**Reply by lovelydinosaur** [reply]
Yeah, please. ☺️
